mono/utils/mono-dl-windows.c

```diff
@@ -68,7 +72,11 @@ mono_dl_open_file (const char *file, int flags)
 		if (!hModule)
 			SetLastError (last_error);
 	} else {
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
```

Comment:
How about GetModuleEx instead? Is that supported? I added it in XP and made it work properly in XPSP1.

Reply:
Though some of the mono/eglib dynamic linking code is wierd anyway, using psapi..

Reply:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms683200(v=vs.85).aspx "Desktop Apps Only"

Reply:
Oh sorry I thought I checked. Ok, and please ask to have it put back.

Reply:
I did, a few years ago. And 2 years ago. And a year ago. Their answer was "no" every time...

Reply:
Stuff keeps getting put back to support existing code. We should keep asking imho unless we really don’t need it.

Reply:
@jaykrell: Is there any action we need here?

Reply:
Since this function mono_dl_open_file now have a lot of variations between Classic Win32 and UWP in it might make sense to write two versions of it, one for classic Win32 kept in this file and one for UWP. The UWP version could then go into mono-dl-windows-uwp.c.

Reply:
@lateralusX: This is a blocking issue for this PR? Or can we wait until we visit this code again to make the new file?

Reply:
Sure, we can fix this later.